book/06-github/sections/2-contributing.asc

```diff
@@ -260,160 +260,161 @@ Isa sa mga magagandang bagay tungkol sa Git ay maaari mong patuloy na gawin iyon
 
-The quotes look something like this:
+Ang mga quote ay magiging tulad nito:
 
 [source,text]
 ----
-> Whether 'tis Nobler in the mind to suffer
-> The Slings and Arrows of outrageous Fortune,
+> Kahit na ito'y Nobler sa isip na magdusa
+> Ang mga Sling at Arrow ng mapangahas na Kapalaran,
 
-How big are these slings and in particular, these arrows?
+Gaano kalaki ang mga sling na ito at sa partikular, ang mga arrow na ito?
 ----
 
-Once rendered, the comment will look like <<_md_quote>>.
+Kapag nagawa, ang komento ay magiging tulad ng <<_md_quote>>.
 
 [[_md_quote]]
-.Rendered quoting example.
+.Halimbawa ng nagawang pag-quote.
 image::images/markdown-05-quote.png[Rendered quoting]
 
 ===== Emoji
 
-Finally, you can also use emoji in your comments. This is actually used quite extensively in comments you see on many GitHub Issues and Pull Requests. There is even an emoji helper in GitHub. If you are typing a comment and you start with a `:` character, an autocompleter will help you find what you're looking for.
+Sa wakas, maaari ka ring gumamit ng emoji sa iyong mga komento. Talagang ginagamit ito nang lubos sa mga komento na nakikita mo sa maraming mga isyu ng GitHub at mga Kahilingan na Pull. Mayroon ding isang emoji helper sa GitHub. Kung nagta-type ka ng komento at nagsimula ka ng isang `:` na karakter, tutulungan ka ng autocompleter na makita kung ano ang iyong hinahanap.
 
 [[_md_emoji_auto]]
 .Emoji autocompleter in action.
 image::images/markdown-06-emoji-complete.png[Emoji autocompleter]
 
-Emojis take the form of `:<name>:` anywhere in the comment. For instance, you could write something like this:
+Ang Emojis ay kumukuha sa form ng `:<name>:` saanman sa komento. Halimbawa, maaari kang magsulat ng ilang bagay tulad nito:
 
 [source,text]
 ----
-I :eyes: that :bug: and I :cold_sweat:.
+Ako ay :eyes: sa :bug: at ako'y :cold_sweat:.
 
-:trophy: for :microscope: it.
+:trophy: para :microscope: ito.
 
-:+1: and :sparkles: on this :ship:, it's :fire::poop:!
+:+1: at :sparkles: sa :ship: na ito, ito'y :fire::poop:!
 
 :clap::tada::panda_face:
 ----
 
-When rendered, it would look something like <<_md_emoji>>.
+Kapag nagawa, ito ay magiging tulad ng <<_md_emoji>>.
 
 [[_md_emoji]]
-.Heavy emoji commenting.
+.Mabibigay na pagkokomento na emoji.
 image::images/markdown-07-emoji.png[Emoji]
 
-Not that this is incredibly useful, but it does add an element of fun and emotion to a medium that is otherwise hard to convey emotion in.
 
+Hindi na ito ay hindi kapani-paniwala kapaki-pakinabang, ngunit ito ay nagdaragdag ng isang elemento ng kasiyahan at damdamin sa isang daluyan na kung hindi man ay mahirap upang ihatid ang damdamin.
 
```

Comment:
"Hindi na ito ay hindi kapani-paniwala" i think it sounds better if we do it like this "Hindi sa ito ay hindi kapani-paniwala"